Kiss.Bff/Extern/ZaakGerichtWerken/Contactmomenten/ContactmomentDetailsRapportageOverzicht.cs

```diff
@@ -38,20 +37,33 @@ public async Task<IActionResult> Get(
                 return BadRequest($"Page size must be between 1 and {MaxPageSize}.");
             }
 
+            var totalCount = await _db.ContactMomentDetails
+                .Where(x => x.Startdatum >= fromDate && x.Startdatum <= toDate)
+                .CountAsync(token);
+
             var contactmomenten = await _db.ContactMomentDetails
                 .Where(x => x.Startdatum >= fromDate && x.Startdatum <= toDate)
                 .OrderByDescending(x => x.Startdatum)
                 .Skip((page - 1) * pageSize)
                 .Take(pageSize)
                 .ToListAsync(token);
 
-            return Ok(contactmomenten);
-        }
+            string? next = (page * pageSize < totalCount)
+                ? $"/api/contactmomentendetails?from={from}&to={to}&pageSize={pageSize}&page={page + 1}"
```

Comment:
lijkt me robuuster als je hier het huidige path gebruikt ipv een string `$"{Request.Path.Value}?from={from}&to={to}&pageSize={pageSize}&page={page + 1}"`
geldt natuurlijk ook voor previous.
maar ehh, moet dit niet de volledige url zijn?

Reply:
aangepast